ros2_orca:
  ros__parameters:
    robot_initial_positions: [
      1.0, -0.75,
      1.0, -1.25,
      1.0, -1.75,
      ]
    goal_positions: [
      -1.0, 1.2,
      -1.0, 1.4,
      -1.0, 1.6,
    ]
    obstacle_positions: [
    -1.0, -0.5,
    0.0, -0.5,
    0.0, 0.5,
    -1.0, 0.5,
    ]
    time_step: 0.5
    num_robots: 3
    radius: 0.2
    deadlockOccurs: 10
    deadlock_threshold: 0.0001
    maxCycle: 2500
    neighborDist: 0.5
    maxNeighbors: 10
    timeHorizon: 1.1
    timeHorizonObst: 1.1
    maxSpeed: 0.3
    # random attack parameters
    randomAttackStart: 0
    randomAttackEnd: 231
    attackerIds: "1"
    numOfFalseMessages: 10
    positionMax: 5
    velocityMax: 2.5
    timeMin: 0
    timeMax: 100
    # falsification parameters
    minPosChange: -5.0
    maxPosChange: 5.0
    minVelChange: -2.5
    maxVelChange: 2.5
    victimRobotId: 2
    attackedRobotId: 2
    deadlockTimestep: 10
    deadlockPosChange: 0.01
    attackType: 1
    numberOfFalseMessage: 10
    falsificationRuns: 1
    falsificationIterations: 1000
    totalTimeStep: 230.50
    delayConstant: 2
    pointX: 7
    pointY: 6
    pointZ: 5
    benign_run: 1

RVO3D:
  ros__parameters:
    obstacle_positions_path: "/home/eastwest/dorca_ws/test_bed/work/ros2_ws/src/RVO2-3D/examples/obstacle_positions.txt"